data/releases/schedule.yaml

```
@@ -4,10 +4,13 @@ schedules:
   maintenanceModeStartDate: 2024-04-28
   endOfLifeDate: 2024-06-28
   next:
-    release: 1.27.1
-    cherryPickDeadline: 2023-05-10
+    release: 1.27.2
+    cherryPickDeadline: 2023-05-12
     targetDate: 2023-05-17
   previousPatches:
+    - release: 1.27.1
+      cherryPickDeadline: ""
+      targetDate: 2023-04-14
```

Let's put a note here that this patch release has been cut due to a regression, for example:
```suggestion
      targetDate: 2023-04-14
      note: >-
        [Regression](https://groups.google.com/g/kubernetes-announce/c/9FTKjmIFOTw/m/TH6cJT64AAAJ)
```

---

Added & rebased/squashed in 8ed90b42bb0ef08e3d1fd906ed8b878953c80aa0